ibm/provider.go

```diff
@@ -948,6 +963,9 @@ func Validator() ValidatorDict {
 				"ibm_scc_si_occurrence":                   resourceIBMSccSiOccurrenceValidator(),
 				"ibm_cbr_zone":                            resourceIBMCbrZoneValidator(),
 				"ibm_cbr_rule":                            resourceIBMCbrRuleValidator(),
+				"ibm_scc_posture_v2_collectors":           resourceIBMSccPostureV2CollectorsValidator(),
```

Comment:
rename resources(remove v2).. why is it all plural? can this resource create multiple collectors, scopes and credentials?
If not please rename to sigular something like below

```
ibm_scc_posture_v2_collectors ==>ibm_scc_posture_collector
ibm_scc_posture_v2_scopes ==>ibm_scc_posture_scope
ibm_scc_posture_v2_credentials ==>ibm_scc_posture_credential
```
please add this near [already existing scc resource, ](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/a0d7985697370c22c1991ae74d4ad8e9c66517fa/ibm/provider.go#L832)
so that all scc resources are grouped at one place